src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const styles = sourceString.split(';');
+
+  styles.forEach((style) => {
+    const [property, value] = style.split(':').map((item) => item.trim());
+
+    if (property && value) {
+      result[property] = value;
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. While `forEach` is an iteration method and is allowed, consider using `reduce` for a more functional approach, which could simplify the function and avoid the need for an external `result` variable.